spec/lib/ansible/runner/credential/network_credential_spec.rb

```diff
@@ -0,0 +1,186 @@
+require 'ansible/runner'
+require 'ansible/runner/credential'
+
+RSpec.describe Ansible::Runner::NetworkCredential do
+  it ".auth_type is the correct Authentication sub-class" do
+    expect(described_class.auth_type).to eq("ManageIQ::Providers::EmbeddedAnsible::AutomationManager::NetworkCredential")
+  end
+
+  context "with a credential object" do
+    around do |example|
+      Dir.mktmpdir("ansible-runner-credential-test") do |dir|
+        @base_dir = dir
+        example.run
+      end
+    end
+
+    let(:auth)     { FactoryBot.create(:embedded_ansible_network_credential, auth_attributes) }
+    let(:cred)     { described_class.new(auth.id, @base_dir) }
+    let(:key_file) { File.join(@base_dir, "env", "network_ssh_key") }
+    let(:auth_attributes) do
+      {
+        :userid   => "manageiq-network",
+        :password => "network_secret"
+      }
+    end
+
+    describe "#command_line" do
+      it "returns an empty hash" do
+        expect(cred.command_line).to eq({})
+      end
+    end
+
+    # Modeled off of awx codebase:
+    #
+    #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/tasks.py#L1432-L1443
+    #
```

Comment:
Does this comment belong in the tests? (expected in the code, but not the tests)

Reply:
I have had it in both in every other credential provider.